Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -2162,6 +2183,50 @@ - (IBAction)showCustomNavigationItemTask:(id)sender {
     [self beginTaskWithIdentifier:CustomNavigationItemTaskIdentifier];
 }
 
+#pragma mark - Review step task
+
+- (id<ORKTask>)makeReviewStepTask {
+    ORKQuestionStep *step1 = [[ORKQuestionStep alloc] initWithIdentifier:@"reviewStepTask.step1"];
+    step1.title = @"step 1 title";
+    step1.text = @"step 1 text";
+    step1.answerFormat = [ORKAnswerFormat booleanAnswerFormat];
+    ORKQuestionStep *step2 = [[ORKQuestionStep alloc] initWithIdentifier:@"reviewStepTask.step2"];
+    step2.title = @"step 2 title";
+    step2.text = @"step 2 text";
+    step2.answerFormat = [ORKAnswerFormat booleanAnswerFormat];
+    ORKReviewStep *reviewStep = [[ORKReviewStep alloc] initWithIdentifier:@"reviewStepTask.reviewStep"];
+    reviewStep.title = @"review step title";
+    reviewStep.text = @"review step text";
+    return [[ORKOrderedTask alloc] initWithIdentifier: ReviewStepTaskIdentifier steps:@[step1, step2, reviewStep]];
+}
+
+- (IBAction)showReviewStepTask:(id)sender {
+    [self beginTaskWithIdentifier:ReviewStepTaskIdentifier];
+}
+
+#pragma mark - Review step task
+
+- (id<ORKTask>)makeReviewStepStandaloneTask {
+    ORKQuestionStep *step1 = [[ORKQuestionStep alloc] initWithIdentifier:@"reviewStepStandaloneTask.step1"];
+    step1.title = @"step 1 title";
+    step1.text = @"step 1 text";
+    step1.answerFormat = [ORKAnswerFormat booleanAnswerFormat];
+    ORKQuestionStep *step2 = [[ORKQuestionStep alloc] initWithIdentifier:@"reviewStepStandaloneTask.step2"];
+    step2.title = @"step 2 title";
+    step2.text = @"step 2 text";
+    step2.answerFormat = [ORKAnswerFormat booleanAnswerFormat];
+    ORKReviewStep *reviewStep = [[ORKReviewStep alloc] initWithIdentifier:@"reviewStepStandaloneTask.reviewStep"
+                                                                    steps:@[step1, step2]
```

Comment:
I think there are 2 szenarions: (1) review step within survey = steps are provided by the task view controller (array is nil) (2) review step standalone = steps are provided upon initialization (array is non nil)